### PR TITLE
#1832: Rename renditions stores config fieldset and add comments

### LINKED ISSUE #1832: Rename renditions stores config fieldset and add comments
### Steps to reproduce (*)
1. Go to Stores -> Configuration -> Advanced -> System
2. Expand "Media Gallery Renditions" fieldset

### Expected result (*)
The fieldset should be renamed from "Media Gallery Renditions" to "Media Gallery Image Size" and comments have to be added according to mockup
image

### PR DESCRIPTION
**Pull request by joweecaquicla**
### Description (*)

This PR will introduce the modifications to renditions stores config fieldset. 
### Fixed Issues (if relevant)

1. Fixes https://github.com/magento/adobe-stock-integration/issues/1832

### Manual testing scenarios (*)

1. Go to Stores -> Configuration -> Advanced -> System
2. "Media Gallery Renditions" should now be "Media Gallery Image Size"
3. Expand "Media Gallery Image Size"
4. Comments should be displayed (see attached screenshot)

Screen Shot 2020-09-15 at 2 18 20 AM

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by joweecaquicla** [comment]
@magento run all tests

File: app/code/Magento/MediaGalleryRenditions/etc/adminhtml/system.xml [hunk 1]
```diff
@@ -9,7 +9,9 @@
     <system>
         <section id="system">
             <group id="media_gallery_renditions" translate="label" type="text" sortOrder="1010" showInDefault="1" showInWebsite="0" showInStore="0">
-                <label>Media Gallery Renditions</label>
+                <label>Media Gallery Image Size</label>
+                <comment>Resize images to improve performance and decrease the file size. When you use an image from Media Gallery on the storefront, the smaller image is generated and placed instead of the original.
+                    Changing these settings will update all generated images.</comment>
                 <field id="width" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
                     <label>Max Width</label>
```
Comment (sivaschenko) [inline app/code/Magento/MediaGalleryRenditions/etc/adminhtml/system.xml]: Please rename the fields and add field comments according to the mockup
```suggestion
                    <label>Maximum Width</label>
```

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @joweecaquicla ! Please see my comment

**Comment by joweecaquicla** [comment]
@magento run all tests

**Review by sivaschenko — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sivaschenko, thank you for the review.
[ENGCOM-8207](https://jira.corp.magento.com/browse/ENGCOM-8207) has been created to process this Pull Request

**Comment by engcom-Bravo** [comment]
:heavy_check_mark: QA Passed

**Before** switching to the PR


**After** switching to the PR


Rendition is covered in several manual testing scenarios. For example in - https://studio.cucumber.io/projects/131313/test-plan/folders/1507933/scenarios/5183135

**Comment by joweecaquicla** [comment]
@magento run all tests

**Pull request merged**